terraso_backend/apps/graphql/schema/__init__.py

```diff
@@ -116,6 +119,7 @@ class Mutations(graphene.ObjectType):
     add_visualization_config = VisualizationConfigAddMutation.Field()
     update_visualization_config = VisualizationConfigUpdateMutation.Field()
     delete_visualization_config = VisualizationConfigDeleteMutation.Field()
+    delete_story_map = StoryMapDeleteMutation.Field()
```

Comment:
question: should there me a StoryMapUpdateMutation?

Reply:
@paulschreiber Handlign files it is not straight forward using GraphQL, so that is why for all the operations that require file uploads we are using direct REST endpoints, and this is why you don't see add and update here since those need file uploads